src/char.cr

```diff
@@ -69,6 +69,38 @@ struct Char
     ord - other.ord
   end
 
+  # Makes a new string by adding *str* to itself *times* times.
+  #
+  # ```
+  # '-' * 10 # => "----------"
+  # ```
+  def *(times count : Int)
```

Comment:
Fair point, although I was thinking about the docs, where the argument name is shown.